### DISCUSSION TITLE
What should I do when applying two models for DDI simulations that use different enzyme databases?

### DISCUSSION THREAD

**Discussion by noxavier** [Open-Systems-Pharmacology/Forum#1583] [Questions & Problems]
For the DDI simulation, I used the models in the OSP-PBPK-Model-Library. However, during the DDI simulation, I found that some models are modeled with different enzyme expression databases, some are RT-PCR, and some are Array.
How should I perform the DDI simulation when I face the models of perpetrator and victim that use different enzyme expression databases?

**Comment by StephanSchaller** [comment]
Dear @noxavier, for the same enzymes, all QUALIFIED models from the [model library](https://github.com/Open-Systems-Pharmacology/OSP-PBPK-Model-Library) use the same enzyme database (e.g. CYP3A4 is always from RT-PCR).
If different Enzymes/Transporters are from different databases, this is not an issue for combining model files.
